<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="questionLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>401</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255,255,255,128); border-radius: 10px; padding: 5px;
</string>
    </property>
    <property name="text">
     <string>Room</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignJustify|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="healthLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="answerLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="submitButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>388</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="puzzleDisplay">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>641</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="roomNameLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
